integreat_cms/summ_ai_api/summ_ai_api_client.py

```diff
@@ -172,3 +215,72 @@ def translate_queryset(self, queryset, language_slug):
         # Commit changes to the database
         for translation_helper in translation_helpers:
             translation_helper.commit(easy_german)
+
+    @classmethod
+    def validate_response(cls, response_data, response_status):
+        """
+        Checks if translated text is found in SummAiApi-response
+
+        :param response_data: The response-data from SummAiApi
+        :type response_data: dict
+
+        :param response_status: The response-status form SummAiApi-Request
+        :type response_status: int
+
+        :returns: True or False
+        :rtype: bool
+
+        :raises SummAiRuntimeError: One of the parameters is not an integer.
+        :raises SummAiInvalidJSONError: Invalid JSON response returned by the API.
```

Comment:
This exception is not actually returned by this function

```suggestion
```

Reply:
Or perhaps you really wanted to raise it instead of SummAiRuntimeError?

Reply:
Trying to recall what happened, seems like I forgot this when raising SummAiInvalidJSONError was moved to outside of and before calling this function in `translate_text_field()` (line 104 currently)